storage/imxrt-flash/flashsrv.c

```diff
@@ -82,9 +84,16 @@ struct {
 	flashsrv_memory_t flash_memories[FLASH_MEMORIES_NO];
 	uint32_t flexspi_addresses[FLASH_MEMORIES_NO];
 	flashsrv_partitionOps_t ops;
+	uint8_t excludingOpen;
 } flashsrv_common;
 
 
+static void usage(const char *progname)
+{
+	ps_log("usage : %s [-o]\n", progname);
+}
```

Comment:
Looks like remaing of some old approach?

Reply:
@gorbanchovie please look at the compilation results before adding anyone to the review - not to waste their time
![image](https://github.com/phoenix-rtos/phoenix-rtos-devices/assets/3939983/4d50b01a-72dd-4c0a-900f-dfa2719badcc)